src/it/java/net/spy/memcached/AutoDiscoveryTest.java

```diff
@@ -28,15 +29,15 @@ public class AutoDiscoveryTest {
   
   @Before
   public void setUp() throws Exception {
-    cf = new DefaultConnectionFactory() {
+    cf = new ClientTestConnectionFactory() {
       @Override
       public ClientMode getClientMode() {
         return TestConfig.getInstance().getClientMode();
       }
       
       @Override
       public long getOperationTimeout() {
-        return 50;
+        return 500;
       }
 
       @Override
```

Comment:
I think you can remove this method as it is already defined in the base test class